### PR TITLE
Update to v6.3

### PR DESCRIPTION
**Pull request by magna-z**

### CODE REVIEW

**Comment by tananaev** [comment]
We need to update Java to 17.

File: Dockerfile.debian [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM openjdk:11-jre-slim-bullseye
```
Comment (tananaev) [inline Dockerfile.debian:1]: This won't work.

File: Dockerfile.ubuntu [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FROM adoptopenjdk:11-jre-hotspot-focal
```
Comment (tananaev) [inline Dockerfile.ubuntu:1]: And this

**Comment by alexandrevannier** [comment]
Yes Java need to be updated, i had to roll back from traccar:latest to traccar:6.2

Error in the logs of my traccar container after the auto update with watchtower 
java.lang.UnsupportedClassVersionError: org/traccar/Main has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0

Thank in advance for the modification

**Pull request merged**